index.html

```diff
@@ -10,7 +10,26 @@
     <!-- шапка сайта  -->
     <header>Шапка сайта</header>
     <!-- секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">

```

Comment:
You need to replace the id value with "download"

Reply:
fixed